src/main/java/bio/terra/cda/app/models/DataSetInfo.java

```diff
@@ -89,7 +89,7 @@ public ColumnDefinition getColumnDefinitionByFieldName(String fieldName, String
     if (fieldName.contains(".")) {
       // it's a mapping field
       String[] parsed = fieldName.split("\\.", 2);
-      TableInfo mappingTable = mappingTableInfoMap.get(parsed[0]);
+      TableInfo mappingTable = getTableInfo(parsed[0]);
       return Arrays.stream(mappingTable.getColumnDefinitions()).filter(col -> col.getName().equals(parsed[1])).findFirst().orElse(null);
```

Comment:
This isn't necessary at all, but I find this code a bit hard to follow. (For me) it would be easier to read if the filter conditional were parsed out into its own variable (if possible) so it read
".filter(filterCondition).findFirst().orElse(null);"

Are there conventions in Java for how to do this otherwise? Like whitespace or formatting?

Reply:
One possibility for clarity would be for the "parsed" values to be more explicit. 
E.g. 
```
String tablename = parsed[0];
String colname = parsed[1];
```

I have seen some Java streams ops that use whitespace for clarity. E.g.:
```
List<Customer> charlesWithMoreThan100Points = customers
  .stream()
  .filter(c -> c.getPoints() > 100 && c.getName().startsWith("Charles"))
  .collect(Collectors.toList());
```

So in this case maybe more like:

```
Arrays.stream(mappingTable.getColumnDefinitions())
    .filter(col->col.getName().equals(parsed[1]))
    .findFirst().orElse(null);
```

Reply:
I like both these suggestions. I think they make it look a lot easier to understand!